src/main/java/com/mitchej123/hodgepodge/config/TweaksConfig.java

```diff
@@ -199,6 +199,11 @@ public class TweaksConfig {
     @Config.DefaultInt(64)
     public static int ic2SeedMaxStackSize;
 
+    @Config.Comment("Ender Quarry RF Storage Override (0 to disable)")
```

Comment:
Maybe mention the original value? Like
```suggestion
    @Config.Comment("Ender Quarry RF Storage Override (ExU default value: 10000000) (0 to use default value)")
```